packages/css/tabs.css

```diff
@@ -23,59 +50,34 @@
   color: var(--dsc-tabs-tab-color);
   position: relative;
   font-family: inherit;
-}
-
-.ds-tabs__content {
-  padding: var(--dsc-tabs-content-padding);
-  color: var(--dsc-tabs-content-color);
-}
-
-.ds-tabs--sm {
-  --dsc-tabs-tab-padding: var(--ds-spacing-2) var(--ds-spacing-4);
-  --dsc-tabs-content-padding: var(--ds-spacing-4);
-}
 
-.ds-tabs--md {
-  --dsc-tabs-tab-padding: var(--ds-spacing-3) var(--ds-spacing-5);
-  --dsc-tabs-content-padding: var(--ds-spacing-5);
-}
-
-.ds-tabs--lg {
-  --dsc-tabs-tab-padding: var(--ds-spacing-4) var(--ds-spacing-6);
-  --dsc-tabs-content-padding: var(--ds-spacing-6);
-}
-
-@media (hover: hover) and (pointer: fine) {
-  .ds-tabs__tab:hover:not([aria-selected='true']) {
-    --dsc-tabs__tab-bottom-border-color: var(--ds-color-neutral-border-subtle);
-    --dsc-tabs-tab-color: var(--ds-color-neutral-text-default);
+  &[aria-selected='true'] {
+    --dsc-tabs__tab-bottom-border-color: var(--ds-color-accent-base-default);
+    --dsc-tabs-tab-color: var(--ds-color-accent-text-subtle);
   }
-}
 
-.ds-tabs__tab[aria-selected='true'] {
-  --dsc-tabs__tab-bottom-border-color: var(--ds-color-accent-base-default);
-  --dsc-tabs-tab-color: var(--ds-color-accent-text-subtle);
-}
+  @composes ds-focus from './utilities.css';
 
-.ds-tabs__tab:focus {
-  z-index: 2;
-}
+  &:focus-visible {
+    z-index: 2;
```

Comment:
Could we just do `position: relative` and also add a comment about why we are doing this? 😊

Reply:
No, we need to make sure it lands on top of the other buttons. Using just position relative makes the active line bleed through the focus indicator

Reply:
This is what happens when we use just `position: relative`:
![image](https://github.com/user-attachments/assets/b54ef1cb-7994-44b0-9fc6-8e0dce1b666a)

Reply:
Right! comment on `z-index` and I'm happy <3